hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl

```diff
@@ -98,5 +98,8 @@ protocol SpanEventViewProtocol {
     array<string> space_ids = [];
 
     int api_exit_calls = 0;
+
+    // map of api callee name -> incoming call count
+    map<string> api_callee_name_count = {};
```

Comment:
Should this be `map<int>`? We should parse the string to integer in the view gens

Reply:
in pinot, we have support for storing only string value for map type. So, wouldn't matter (originally I had it as integer)?

Reply:
I am not sure it is supported at pinot level, we can check by locally trying change with docker-compose setup.

Reply:
I tried it out, 
1. For docker compose it works fine, I used 0 as default null value for the column corresponding to the map value. 
2. However we use view-creation job to create pinot views, now the view-creation framework had some assumptions around dataType of map value. Tried a fix here https://github.com/hypertrace/view-generator-framework/pull/28 

Query service will read the key and values into a map<string, string> https://github.com/hypertrace/query-service/blob/main/query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/PinotBasedRequestHandler.java#L537, but that should be fine.
cc @laxmanchekka

Reply:
Going ahead with map<string, string> for this change, will explore storing data as JSONDoc https://docs.pinot.apache.org/users/user-guide-query/supported-transformations subsequently.